domains/transfer-agreements/TransferAgreements.API/API/ApiModels/Requests/CreateTransferAgreementValidator.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using FluentValidation;
+
+namespace API.ApiModels.Requests
+{
+    public class CreateTransferAgreementValidator : AbstractValidator<CreateTransferAgreement>
+    {
+        public CreateTransferAgreementValidator(string senderTin)
+        {
+            var now = DateTimeOffset.UtcNow;
+
+            RuleFor(t => t.StartDate)
+                .NotEmpty()
+                .WithMessage("Start Date cannot be empty.")
+                .GreaterThanOrEqualTo(_ => now.ToUnixTimeSeconds())
+                .WithMessage("Start Date cannot be in the past.")
+                .MustBeBeforeYear10000();
+
+            RuleFor(t => t.EndDate)
```

Comment:
Here you name the reference to CreateTransferAgreement `t` and in line `21` you call it dto, can you please rename to `createTransferAgreement`?

Reply:
This goes for all the `RuleFor`